test/onnx/exporter/test_small_models_e2e.py

```diff
@@ -58,6 +58,76 @@ def forward(self, x):
         onnx_testing.assert_onnx_program(onnx_program)
         self.assertNotIn("Cast", [node.op_type for node in onnx_program.model.graph])
 
+    def test_onnx_export_control_flow(self):
+        class CondModel(torch.nn.Module):
+            def forward(self, x):
+                def true_fn(x):
+                    return x + 1.0
+
+                def false_fn(x):
+                    return x - 42.0
+
+                y = torch.cond(x.sum() > 0, true_fn, false_fn, [x])
+                return y
+
+        onnx_program = torch.onnx.export(
+            CondModel(),
+            (torch.tensor([1, 2]),),
+            dynamo=True,
+            fallback=False,
+        )
+        onnx_model = onnx_program.model
+        self.assertIn(
```

Comment:
nit: why not test that the main graph has a `If` node? Instead of hard-coding names as down below (which may change without breaking things).

Reply:
Thanks! Done